flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/serializer/schema/ColumnSerializer.java

```diff
@@ -46,6 +46,19 @@ public class ColumnSerializer extends TypeSerializerSingleton<Column> {
     private final MetadataColumnSerializer metadataColumnSerializer =
             MetadataColumnSerializer.INSTANCE;
 
+    /**
+     * Identify for different major version:
+     * for Version 3.0/3.1 is 0,
+     * for version 3.2 is 1
+     */
```

Comment:
This seems not quite accurate, since 3.1.1 has bumped `SchemaSerializer` version from 0 to 1 and adds `partitionKey` options. Since default value will not be shipped until 3.2, should we choose schema-related serialization version?

Reply:
done.